Community: Add Chakra Discord Channel to Documentation Nav

**Is your feature request related to a problem? Please describe.**
In order give community members a place to discuss and get more interactive communications around solutions/problems/discussions in Chakra UI, a more asynchronous place is more suitable than issues.

**Describe the solution you'd like**
Open up Chakra Discord and include it in Docs Navbar

Hi @codebender828 , I ran into the error `Error: Cannot find module '@chakra-ui/vue'. Please verify that the package.json has a valid "main" entry  ` during initial setup. 
I have explained the whole thing in the Discord Vue Documentation channel.
Please have a look and guide on how to proceed.
Thankyou.

--- a/.storybook/config.js
+++ b/.storybook/config.js
@@ -33,6 +33,10 @@ import {
   faGithub
 } from '@fortawesome/free-brands-svg-icons'
 
+import {
+  faDiscord
+} from '@fortawesome/free-brands-svg-icons'
+
 Vue.use(Chakra, {
   icons: {
     iconPack: 'fa',
@@ -51,6 +55,7 @@ Vue.use(Chakra, {
       faSearch,
       faTimesCircle,
       faGithub,
+      faDiscord,
       faCheckCircle,
       faCog,
       faLock,

--- a/website/components/Navbar.vue
+++ b/website/components/Navbar.vue
@@ -43,6 +43,17 @@
           icon="github"
         />
       </CBox>
+      <CBox as="li" mr="2">
+        <CIconButton
+          as="a"
+          variant="ghost"
+          variant-color="gray"
+          aria-label="Join Discord channel"
+          target="_blank"
+          href="https://discord.gg/sq2Kp6x"
+          icon="discord"
+        />
+      </CBox>
       <CBox as="li">
         <CIconButton
           v-chakra="{

--- a/website/plugins/chakra-ui.js
+++ b/website/plugins/chakra-ui.js
@@ -1,7 +1,7 @@
 import Vue from 'vue'
 import * as Chakra from '@chakra-ui/vue'
 import theme from '@chakra-ui/theme-vue'
-import { faGithub, faTwitter, faLinkedin } from '@fortawesome/free-brands-svg-icons'
+import { faGithub, faDiscord, faTwitter, faLinkedin } from '@fortawesome/free-brands-svg-icons'
 import {
   faBolt,
   faAnchor,
@@ -38,6 +38,7 @@ Vue.use(Chakra.default, {
     iconPack: 'fa',
     iconSet: {
       faGithub,
+      faDiscord,
       faAnchor,
       faBolt,
       faUniversalAccess,

